### PR TITLE
site: edited installation page and switched Download link to point at it

### PR DESCRIPTION
**Pull request by jkowalski**
Emphasized the use of package managers to download and keep Kopia up-to-date.
Added instructions for using Scoop on Windows.

### CODE REVIEW

File: site/content/docs/Installation/_index.md [hunk 1]
```diff
@@ -4,49 +4,52 @@ linkTitle: "Installation"
 weight: 20
 ---
 
-### Download
+### Installing Kopia 
 
-Kopia is released as a single binary, available for many operating systems:
+Kopia in an open source software (OSS) developed by a community on GitHub.
```
Comment (julio-lopez) [inline site/content/docs/Installation/_index.md:9]:
```suggestion
Kopia is an open source software (OSS) developed by a community on GitHub.
```

File: site/content/docs/Installation/_index.md [hunk 2]
```diff
@@ -4,49 +4,52 @@ linkTitle: "Installation"
...
+Kopia is available in two variants:
 
-```shell
-# Import official signing key
-$ curl https://kopia.io/signing-key | gpg --import -
+* `Command Line Interface (CLI)` which is a stand-alone binary called `kopia` and which can be used a terminal window or scripts. This is typically the preferred option for power users, system administrators, etc.
 
-# Verify that file checksums are ok
-$ sha256sum --check checksums.txt
+* `Graphical User Interface (GUI)` called `KopiaUI` which is a desktop application that offers graphical user interface tailored towards less technical users. It has fewer features than CLI but
```
Comment (julio-lopez) [inline site/content/docs/Installation/_index.md:19]: This sentence seems to be incomplete.

**Review by julio-lopez — COMMENTED** [review]
Minor comments, @ntolia may offer additional suggestions, especially for the wording around community development.

File: site/content/docs/Installation/_index.md (hunk 2 above)
Comment (jkowalski) [reply site/content/docs/Installation/_index.md]: removed.

File: site/content/docs/Installation/_index.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Kopia is available in two variants:
 
 * `Command Line Interface (CLI)` which is a stand-alone binary called `kopia` and which can be used a terminal window or scripts. This is typically the preferred option for power users, system administrators, etc.
 
-* `Graphical User Interface (GUI)` called `KopiaUI` which is a desktop application that offers graphical user interface tailored towards less technical users. It has fewer features than CLI but 
+* `Graphical User Interface (GUI)` called `KopiaUI` which is a desktop application that offers graphical user interface tailored towards less technical users.
```
Comment (julio-lopez) [inline site/content/docs/Installation/_index.md:19]:
```suggestion
* `Graphical User Interface (GUI)`: It is a desktop application, called `KopiaUI`, that offers a friendly user interface.
```
May want to tweak the previous line to match this one.

**Comment by julio-lopez** [comment]
Any additional comments @ntolia may have can be addressed afterwards.

**Pull request merged**